.github/todo.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Since this was done before the vue workshops, I will assume you have now learnt and are using single page components, as a .vue file.